CoreLayersView/src/au/gov/asd/tac/constellation/views/layers/docs/layers-view.html

```diff
@@ -11,16 +11,16 @@ <h1>Layers View</h1>
             <caption>Layers View Actions</caption>
             <tr>
                 <th scope="col">
-                    <strong>Constellation Action</strong>
+                    "Constellation Action"
```

Comment:
I think the double quotes can be removed in the table headers (my bad for not explaining for not explaining what I did with strong tags well enough)